sklearn/compose/_column_transformer.py

```diff
@@ -80,6 +81,9 @@ class ColumnTransformer(_BaseComposition, TransformerMixin):
         By setting ``remainder`` to be an estimator, the remaining
         non-specified columns will use the ``remainder`` estimator. The
         estimator must support :term:`fit` and :term:`transform`.
+        If columns in `transformers` are provided as strings, the ordering
```

Comment:
How about "Note that using this feature requires that the DataFrame columns input at fit and transform have identical order."

Reply:
Much better, will replace.